OmsAgent/watcherutil.py

```diff
@@ -74,55 +163,55 @@ def write_waagent_event(self, event):
 
     def create_telemetry_event(self, operation, operation_success, message, duration):
         template = """ {{
-    		"eventId": 1,
-    		"providerId": "69B669B9-4AF8-4C50-BDC4-6006FA76E975",
-    		"parameters": [
+            "eventId": 1,
+            "providerId": "69B669B9-4AF8-4C50-BDC4-6006FA76E975",
+            "parameters": [
                         {{
-            			"name": "Name",
-            			"value": "Microsoft.EnterpriseCloud.Monitoring.OmsAgentForLinux"
-        		}},
+                        "name": "Name",
+                        "value": "Microsoft.EnterpriseCloud.Monitoring.OmsAgentForLinux"
+                }},
                         {{
+
             			"name": "Version",
             			"value": "1.7.3"
-        		}},
+             		}},
+
                         {{
-            			"name": "Operation",
-            			"value": "{}"
-        		}},
+                        "name": "Operation",
+                        "value": "{}"
+                }},
                         {{
-            			"name": "OperationSuccess",
-            			"value": {}
-        		}},
+                        "name": "OperationSuccess",
+                        "value": {}
+                }},
                         {{
-            			"name": "Message",
-            			"value": "{}"
-        		}},
+                        "name": "Message",
+                        "value": "{}"
+                }},
                         {{
-            			"name": "Duration",
-            			"value": {}
-        		}}
-    		]
+                        "name": "Duration",
+                        "value": {}
+                }}
+            ]
             }}"""
 
-        operation_success_as_string = str(operation_success).lower()
-        formatted_message = message.replace("\n", "\\n").replace("\t", "\\t").replace('"', '\"')
-
-        return template.format(operation, operation_success_as_string, formatted_message, duration)
+        return template.format(operation, operation_success, message, duration)
 
     def upload_telemetry(self):
         status_files = [
                 "/var/opt/microsoft/omsagent/log/ODSIngestion.status",
                 "/var/opt/microsoft/omsagent/log/ODSIngestionBlob.status",
                 "/var/opt/microsoft/omsagent/log/ODSIngestionAPI.status",
-                "/var/opt/microsoft/omsconfig/status/dscperformconsistency",
+                "/var/opt/microsoft/omsconfig/status/dscperformrequiredconfigurationchecks",
                 "/var/opt/microsoft/omsconfig/status/dscperforminventory",
-                "/var/opt/microsoft/omsconfig/status/dscsetlcm"
-            ]
+                "/var/opt/microsoft/omsconfig/status/dscsetdsclocalconfigurationmanager"
```

Comment:
These file names should not be changed,

Reply:
I think some mistake in my merge, i will correct those, thanks